docs/source/developing_with_sdk/executing_packaged_app_locally.md

```diff
@@ -13,15 +13,15 @@ monai-deploy run --help
 Output:
 
 ```bash
-usage: monai-deploy run [-h] [-l {DEBUG,INFO,WARN,ERROR,CRITICAL}] [-q] <map-image[:tag]> <input> <output>
+usage: monai-deploy run [-h] [-l <LEVEL>G,INFO,WARN,ERROR,CRITICAL}] [-q] <map-image[:tag]> <input> <output>
```

Comment:
Is this intended, `<LEVEL>G`?

Reply:
You seem to have hawk eyes :)

Let me fix the typo.